docker-compose.yml

```diff
@@ -58,8 +54,6 @@ services:
     networks:
       - main
     restart: on-failure
-    volumes:
```

Comment:
Ditto. I'll need to fix these before it's safe to merge

Reply:
dropped the commit that made this change. should have watched a bit more closely when cherry-picking